bulk-user-setup/bulk-user-setup.config

```diff
@@ -47,13 +55,25 @@ NORMAL=$(tput sgr0)
 ###########################################
 
 #'|' delimited string of roles. These roles will be added by default for all caseworkers
-DEFAULT_ROLES="caseworker"
+#DEFAULT_ROLES="caseworker"
```

Comment:
We can remove the unused global variables

Reply:
Dones